resources/test.json

```diff
@@ -54,6 +54,20 @@
             "test": "paratest --version",
             "tags": ["test"]
         },
+        {
+            "name": "pest",
+            "summary": "an elegant PHP testing Framework with a focus on simplicity, meticulously designed to bring back the joy of testing in PHP",
+            "website": "https://github.com/pestphp/pest",
+            "command": {
+                "composer-bin-plugin": {
+                    "package": "laravel/pest",
```

Comment:
There's no composer package of this name: https://packagist.org/search/?q=laravel/pest&reason=package_not_found